### PR TITLE
Add accessibility mention to PR template

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -26,4 +26,5 @@ _Please, go through these checks before submitting the PR._
 - [ ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
 - [ ] You have commented your code, particularly in hard-to-understand areas
 - [ ] You have performed a self-review of your own code
-- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
\ No newline at end of file
+- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
+- [ ] UI Changes: they satisfy Android accessibility test https://developer.android.com/guide/topics/ui/accessibility
```
Comment (david-allison-1) [inline .github/pull_request_template.md:30]:
```suggestion
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)
```

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Discussed on Discord - I didn't feel the landing page was useful (information overload), a concrete action is better and much less intimidating.

EDIT: But! I'm for the change, just feel we can improve the wording

**Comment by Arthur-Milchior** [comment]
Accepting David's version

**Review by david-allison-1 — APPROVED** [review]

**Comment by mikehardy** [comment]
Love this!

**Comment by mikehardy** [comment]
Hi there!

Just a friendly notice (one per person for PRs merged in March, regardless of PR quantity) that we try to process OpenCollective payments monthly - it's time for March 2021 submissions

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

Thanks!

**Pull request merged**